src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs

```diff
@@ -220,8 +224,11 @@ public override IEnumerable ItemsSource
 
         /// <summary>
         /// DataTemplate for items.
+        /// Create visual contents and binding properties.
+        /// return object type is restricted RecyclerViewItem.
+        /// @see #DataTemplate
```

Comment:
It looks like doxygen syntax SEE, in C# it's different
```suggestion
        /// <seealso cref="Tizen.NUI.Binding.DataTemplate"/>
```